stores/src/main/java/com/issoft/training/stores/comparaton/ProductComparator.java

```diff
@@ -0,0 +1,45 @@
+package com.issoft.training.stores.comparaton;
+
+import com.issoft.training.domain.shop.Product;
+
+import java.util.Comparator;
+
+public class ProductComparator implements Comparator<Product> {
+
+    private final String orderingName;
+    private final String orderingPrice;
+    private final String orderingRate;
+
+    public ProductComparator(String orderingName, String orderingPrice, String orderingRate) {
```

Comment:
what if we append one more attribute to product and sorting, please try to think about how to refactor this class.